tests/integration/components/convert-or-copy-project-test.js

```diff
@@ -32,20 +32,20 @@ test('choosing copy changes mode to copy', function(assert){
     assert.equal(this.get('convertOrCopy'), 'copy');
 });
 
-test('choosing copy makes nodeTitle null and requires a title change', function(assert){
-    this.set('nodeTitle', 'Como nossos pais');
+test('choosing copy makes the preprint title null and requires a title change', function(assert){
+    this.set('title', 'Como nossos pais');
     this.set('titleValid', true);
 
     this.render(hbs`{{convert-or-copy-project
         clearDownstreamFields=(action noop)
         nextUploadSection=(action noop)
-        nodeTitle=nodeTitle titleValid=titleValid
+        title=title titleValid=titleValid
     }}`);
 
     this.$('#copy').click();
 
     assert.ok(!this.get('titleValid'));
-    assert.equal(this.get('nodeTitle'), null);
+    assert.equal(this.get('title'), null);
 });
 
 test('choosing convert displays converting options and changes mode to convert', function(assert){
```

Comment:
while you're in there, space after { here too. I bet there are more in this file.